test/jdk/java/awt/print/PrinterJob/PrintTextTest.java

```diff
@@ -21,193 +21,212 @@
+        name = "Page " + page++;
+        pt = new PrintText(name, font, scaleTx, false, preferredSize);
+        pane.add(name, pt);
+        book.append(pt, portrait);
+        book.append(pt, landscape);
 
         font = new Font("Dialog", Font.PLAIN, 18);
         scaleTx = AffineTransform.getScaleInstance(-1.25, 1.25);
         scaleTx.translate(-preferredSize/1.25, 0);
-        name = "Page " + new Integer(page++);
-        ptt = new PrintTextTest(name, font, scaleTx, false);
-        p.add(name, ptt);
-        book.append(ptt, portrait);
-        book.append(ptt, landscape);
+        name = "Page " + page++;
+        pt = new PrintText(name, font, scaleTx, false, preferredSize);
+        pane.add(name, pt);
+        book.append(pt, portrait);
+        book.append(pt, landscape);
 
         font = new Font("Dialog", Font.PLAIN, 18);
         scaleTx = AffineTransform.getScaleInstance(1.25, -1.25);
         scaleTx.translate(0, -preferredSize/1.25);
-        name = "Page " + new Integer(page++);
-        ptt = new PrintTextTest(name, font, scaleTx, false);
-        p.add(name, ptt);
-        book.append(ptt, portrait);
-        book.append(ptt, landscape);
+        name = "Page " + page++;
+        pt = new PrintText(name, font, scaleTx, false, preferredSize);
+        pane.add(name, pt);
+        book.append(pt, portrait);
+        book.append(pt, landscape);
 
         font = font.deriveFont(rotTx);
-        name = "Page " + new Integer(page++);
-        ptt = new PrintTextTest(name, font, null, false);
-        p.add(ptt, BorderLayout.CENTER);
-        p.add(name, ptt);
-        book.append(ptt, portrait);
-        book.append(ptt, landscape);
+        name = "Page " + page++;
+        pt = new PrintText(name, font, null, false, preferredSize);
+        pane.add(pt, BorderLayout.CENTER);
+        pane.add(name, pt);
+        book.append(pt, portrait);
+        book.append(pt, landscape);
 
         font = new Font("Monospaced", Font.PLAIN, 12);
-        name = "Page " + new Integer(page++);
-        ptt = new PrintTextTest(name, font, null, false);
-        p.add(ptt, BorderLayout.CENTER);
-        p.add(name, ptt);
-        book.append(ptt, portrait);
-        book.append(ptt, landscape);
+        name = "Page " + page++;
+        pt = new PrintText(name, font, null, false, preferredSize);
+        pane.add(pt, BorderLayout.CENTER);
+        pane.add(name, pt);
+        book.append(pt, portrait);
+        book.append(pt, landscape);
 
         Font xfont = font.deriveFont(AffineTransform.getScaleInstance(1.5, 1));
-        name = "Page " + new Integer(page++);
-        ptt = new PrintTextTest(name, xfont, null, false);
-        p.add(ptt, BorderLayout.CENTER);
-        p.add(name, ptt);
-        book.append(ptt, portrait);
-        book.append(ptt, landscape);
+        name = "Page " + page++;
+        pt = new PrintText(name, xfont, null, false, preferredSize);
+        pane.add(pt, BorderLayout.CENTER);
+        pane.add(name, pt);
```

Comment:
```suggestion
        pane.addTab(name, pt);
```
And the following cases.